src/observer.js

```diff
@@ -177,17 +157,12 @@ function makeComponentReactive(render) {
      * If props are shallowly modified, react will render anyway,
      * so atom.reportChanged() should not result in yet another re-render
      */
-    let skipRender = false
+    this[skipRenderKey] = false
     /**
      * forceUpdate will re-assign this.props. We don't want that to cause a loop,
      * so detect these changes
      */
-    let isForcingUpdate = false
-
-    // make this.props an observable reference, see #124
-    makePropertyObservableReference.call(this, "props")
-    // make state an observable reference
-    makePropertyObservableReference.call(this, "state")
+    this[isForcingUpdateKey] = false
```

Comment:
idem

Reply:
Please clarify, dont understood :(

Reply:
Sorry, making this a non-enumerable prop as well :)